personal_graph/embeddings.py

```diff
@@ -14,13 +13,9 @@ def get_embedding(self, text: str) -> list[float]:
 
 
 class OpenAIEmbeddingsModel(EmbeddingsModel):
-    def __init__(self) -> None:
-        self.client = (
-            OpenAI(api_key=os.getenv("OPEN_API_KEY"))
-            if os.getenv("OPEN_API_KEY")
-            else None
-        )
-        self.model = "text-embedding-3-small"
+    def __init__(self, embed_client: Any, embed_model: str) -> None:
+        self.client = embed_client if embed_client else None
+        self.model = embed_model
 
     def get_embedding(self, text: str) -> list[float]:
         if self.client is None:
```

Comment:
`dimensions=384` should be an arg from the library user as well as it depends on the embedding model being used

Reply:
Done